modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/ExtensionDefinitionParser.java

```diff
@@ -710,6 +711,11 @@ protected ClassLoader getContextClassLoader() {
     return resolver;
   }
 
+  private boolean isRequiredByExclusiveOptional(Set<ModelProperty> modelProperties) {
+    return modelProperties.stream().filter(modelProperty -> modelProperty instanceof ExclusiveOptionalModelProperty
+        && ((ExclusiveOptionalModelProperty) modelProperty).isOneRequired()).count() != 0;
```

Comment:
this could be replaced with ```modelProperties.stream().anyMatch(modelProperty -> modelProperty instanceof ExclusiveOptionalModelProperty  && ((ExclusiveOptionalModelProperty) modelProperty).isOneRequired());```